src/test/java/com/google/devtools/build/lib/runtime/CommandLinePathFactoryTest.java

```diff
@@ -99,6 +100,9 @@ public void createWithNamedRoot() throws Exception {
         .isEqualTo(filesystem.getPath("/path/to/output/base/foo"));
     assertThat(factory.create(ImmutableMap.of(), "%output_base%/foo/bar"))
         .isEqualTo(filesystem.getPath("/path/to/output/base/foo/bar"));
+
+    assertThat(factory.create(ImmutableMap.of(), "%workspace%//foo//bar"))
```

Comment:
Why allow this?

Reply:
It seems gratuitous to disallow double-slashes when most other command-line utilities handle them correctly; in addition, we get it for free through `PathFragment.create`. We could even avoid the edge case by converting the whole string to a `PathFragment` before regex-matching the first segment, but I wanted to do the bare minimum to avoid complicating the cherry pick.

Reply:
Fair enough